sw/device/BUILD

```diff
@@ -0,0 +1,10 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
```

Comment:
Is `cc_library` even right here? Shouldn't this be like, a filegroup?

Reply:
I _think_ I tried to put these into a `filegroup` when I wrote the original demo.  My recollection is that filegroup didn't work and I had to specify linker scripts (and script fragments) as the `deps` of a `cc_library` rule.

Reply:
My recollection is that `filegroup` isn't the right kind of provider for CC dependencies and the linker scripts need to go into the `deps` section of a `cc_*` rule.

Reply:
Ugh. Fine. Bazel isn't great at dealing with linker scripts honestly.

On some level I feel like `info_sections.ld` wants to go in `hdrs` since that's morally what it is... idk

Reply:
Heh.  I think it somehow belongs in `linkopts`.

Reply:
it looks like bazel prefers linker scripts to get handled as deps (at least from what I can see from discussions in 2015.)